src/QuadForm/Quad/ZGenus.jl

```diff
@@ -2669,13 +2670,7 @@ function represents(G1::ZZGenus, G2::ZZGenus)
     return false
   end
 
-  primes = [prime(s) for s in local_symbols(G1)]
-  for s in local_symbols(G2)
-    p = prime(s)
-    if !(p in primes)
-      push!(primes, p )
-    end
-  end
+  primes = union!(Hecke.primes(G1), Hecke.primes(G2))
```

Comment:
Does `primes(G1)` make a copy? If not this is wrong

Reply:
Yes.